### DISCUSSION TITLE
京都大學超級電腦備份錯誤

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#282] [General]
看了這個新聞突然想到，隨著科技與學術發展，我們需要儲存的資料只會越來越多，而屬不清的數據資料只能仰賴電腦儲存，一旦哪一部電腦又遇到了類似的情形，一次的損失將會難以估計，看來資料儲存至少到現在都還沒有萬無一失的辦法。
[https://www.ithome.com.tw/news/148672](https://www.ithome.com.tw/news/148672)

**Comment by 109302003** [comment]
備份也會錯誤讓人好害怕！

**Comment by ghost** [comment]
總覺得如果未來生活越來越科技化，都離不開電腦，哪天點腦系統出現大故障，會產生非常嚴重的問題，從已經默認習慣身邊的一切由電腦自動控制突然什麼都亂了

**Comment by 20010906** [comment]
希望未來能找到解決辦法，阻止憾事發生。

**Comment by chupochun** [comment]
有時候真的不能太過相信電腦

**Comment by yucheng1028** [comment]
這麼多學術資料遺失真的很可惜，或許現代人需要找更安全的資料處理方式

**Comment by ThomasMeng0515** [comment]
現代太過於依賴將資料上傳到雲端，當雲端資料庫出問題的時候，將會造成許多重要的資料遺失，值得我們考慮儲存重要資料的位置。

**Comment by pekoranoinu** [comment]
應該要開發一個專門存放重要資料的資料庫。

**Comment by awkward-willy** [comment]
定期備份已經是未來人人必做的一件事了
不過備份過程中的資料遺失還真的是防不勝防
期望未來能有更好更安穩的技術能做好備份與存放的工作
畢竟未來資料量越來越龐大的情況下
這一定是不可或缺的

**Comment by Lai27330** [comment]
資料管理真的很不容易，不但要定期備份，還要適時將不需要的檔案刪除以免佔用空間，而且當資料一多時部分無法上傳或遺失的情況更難及時發現，希望未來能有更妥善的資料存放方法。

**Comment by treeann** [comment]
超級電腦也會被份錯誤?太誇張了拉

**Comment by etin2468** [comment]
大學的資料備份錯誤喔，那還真是嚴重

**Comment by 110409538** [comment]
我覺得這就像是把所有雞蛋都放在一個籃子裡.

**Comment by Yuhsin000** [comment]
乍看之下還以為是bookroll發生什麼事了，幸好點進去看不是，微積分還指望用bookroll複習呢

**Comment by yi-1111** [comment]
平時做好備份，不怕一時之需

**Comment by sys511613** [comment]
所以說真的好做好備份，如果擔心電腦壞掉的話可以在旁邊放包綠色乖乖

**Comment by leon10456** [comment]
備份也會錯誤 怕爆
